src/components/navbar/navbar.tsx

```diff
@@ -1,53 +1,62 @@
-import {Element, Component, Host, h, Prop, State} from "@stencil/core";
+import { Component, Element, h, Host, Prop, State } from '@stencil/core'
 
 @Component({
-  tag: "bal-navbar",
-  styleUrl: "navbar.scss",
+  tag: 'bal-navbar',
+  styleUrl: 'navbar.scss',
 })
 export class Navbar {
 
-  hasNavbarStartSlot: boolean;
-  hasNavbarEndSlot: boolean;
+  hasNavbarStartSlot: boolean
+  hasNavbarEndSlot: boolean
 
-  @State() isMenuActive: boolean = false;
+  @State() isMenuActive: boolean = false
 
-  @Prop() light = false;
-  @Prop() logoHref = "https://bulma.io";
+  @Prop() light = false
+  @Prop() logoHref = ''
 
-  @Element() el: HTMLElement;;
+  @Element() el: HTMLElement
+;
 
   componentWillLoad() {
-    this.hasNavbarStartSlot = !!this.el.querySelector('[slot="navbar-start"]');
-    this.hasNavbarEndSlot = !!this.el.querySelector('[slot="navbar-end"]');
+    this.hasNavbarStartSlot = !!this.el.querySelector('[slot="navbar-start"]')
+    this.hasNavbarEndSlot = !!this.el.querySelector('[slot="navbar-end"]')
   }
 
   async toggle(): Promise<void> {
-    this.isMenuActive = !this.isMenuActive;
+    this.isMenuActive = !this.isMenuActive
   }
 
   render() {
     return (
       <Host>
-        <nav class={ "navbar is-spaced" + (this.light ? " is-white bal-track-line" : " is-info") }
-             role="navigation" aria-label="main navigation">
+        <nav class={'navbar is-spaced' + (this.light ? ' is-white' : ' is-info')}
+             role="navigation"
+             aria-label="main navigation">
+          <div class="bal-track-line"
+               style={{
+                 position: 'absolute',
+                 top: '0',
+                 left: '0',
+                 display: !this.light ? 'none' : '',
```

Comment:
why even render the `div` if `!this.light`?

Reply:
this is the baloise track line we need that for the light style

Reply:
in the normal mode we hide it